lib/jetstream/api/kv/watcher.ex

```diff
@@ -30,6 +30,16 @@ defmodule Jetstream.API.KV.Watcher do
     GenServer.stop(pid)
   end
 
+  @spec init(keyword) ::
```

Comment:
Was this typespec intentionally added? I think the overall structure of this function is well understood as part of the GenServer API. If we want to document the shape of the state used in this GenServer I would suggest that we provide some more useful type hints like `bucket_name: String.t()` rather than `any()`

Reply:
Oh, no, that was not my intention. My Emacs offers a clickable link to add typespecs, and I clicked on it by accident. I'll remove it.